gazebo_plugins/include/gazebo_plugins/gazebo_ros_camera_utils.h

```diff
@@ -139,8 +140,14 @@ namespace gazebo
     protected: common::Time last_info_update_time_;
 
     /// \brief ROS frame transform name to use in the image message header.
-    ///        This should typically match the link name the sensor is attached.
+    ///        This should typically match the link name the parent sensor + "_optical".
     protected: std::string frame_name_;
```

Comment:
It looks like the documentation for `frame_name_` and `frame_name_optical_` is mixed up